axelrod/tests/unit/test_match_generator.py

```diff
@@ -44,7 +44,7 @@ def test_len(self):
         with self.assertRaises(NotImplementedError):
             len(tt)
 
-    def test_build_match_params(self):
+    def _params(self):
```

Comment:
you seem to have lost `test_build_match_`. I expect this test was not running (always make sure your test fails before it passes :)).